pysql_beam/examples/.ipynb_checkpoints/cloud_sql_to_file-checkpoint.py

```diff
@@ -0,0 +1,109 @@
+"""
+This read some data from cloud sql mysql database
+and write to file
+
+Command to run this script:
+
+python cloud_sql_to_file.py --host localhost --port 3306 --database SECRET_DATABASE \
+--username SECRET_USER --password SECRET_PASSWORD --table YOUR_TABLE --output YOUR_OUTPUT_FLLE
+
+
+For postgres sql:
+python cloud_sql_to_file.py  --host localhost  --port 5432 --database SECRET_DATABASE \
+--username SECRET_USER --password SECRET_PASSWORD --table YOUR_TABLE --output YOUR_OUTPUT_FLLE
+
+"""
+import sys
+import json
+sys.path.insert(0, '/home/jupyter/shapiro-johannes/pysql-beam/pysql-beam')
+sys.path.insert(0, '/home/jupyter/shapiro-johannes/pysql-beam')
+import logging
+import apache_beam as beam
+from pysql_beam.sql_io.sql import SQLSource, SQLWriter, ReadFromSQL
+from pysql_beam.sql_io.wrapper import MySQLWrapper, MSSQLWrapper, PostgresWrapper
+from apache_beam.options.pipeline_options import PipelineOptions, SetupOptions, GoogleCloudOptions
+from apache_beam.io.gcp.internal.clients import bigquery
+from apache_beam.io.gcp.bigquery import WriteToBigQuery, BigQueryDisposition
+
+def log(row, level="debug"):
+    getattr(logging, level.lower())(row)
+    return row
+
+
+class SQLOptions(PipelineOptions):
+
+    @classmethod
+    def _add_argparse_args(cls, parser):
+        # parser.add_value_provider_argument('--host', dest='host', required=False)
+        # parser.add_value_provider_argument('--port', dest='port', required=False)
+        # parser.add_value_provider_argument('--database', dest='database', required=False)
+        # parser.add_value_provider_argument('--table', dest='table', required=False)
+        # parser.add_value_provider_argument('--query', dest='query', required=False)
+        # parser.add_value_provider_argument('--username', dest='username', required=False)
+        # parser.add_value_provider_argument('--password', dest='password', required=False)
+        #parser.add_value_provider_argument('--db_type', dest='db_type', default="mssql", required=False, help="the type of database; allowed are 'mssql', 'mysql' and 'postgres'")
+        parser.add_value_provider_argument('--host', dest='host', default="localhost")
+        parser.add_value_provider_argument('--port', dest='port', default="3306")
+        parser.add_value_provider_argument('--database', dest='database', default="dverma")
+        parser.add_value_provider_argument('--query', dest='query', default="SELECT * FROM dverma.userPointsLedger;")
```

Comment:
Might be remove dverma database value from here and use something which is not user specific!